packages/generator/templates/app/app/blitz-server.ts

```diff
@@ -2,7 +2,7 @@ import { setupBlitzServer } from '@blitzjs/next';
 import { AuthServerPlugin, PrismaStorage } from '@blitzjs/auth';
 import { simpleRolesIsAuthorized } from '@blitzjs/auth';
 import { BlitzLogger } from 'blitz';
-import db from 'db';
+import db from '../db';
```

Comment:
I don't understand why this change was needed. It should work out of the box — if it isn't then something is wrong with our paths setup.

Reply:
Nothing wrong with the path setup. The issue is with the generator cli command and `esbuild-register`. Can't seem to find db when generating a model in a javascript project (not typescript).

Any advice?

Reply:
Hey @dillonraphael, ig this problem can be avoided by removing the dependency on `esbuild` to get the config, instead parsing the file using `recast` and `jscodeshift`. I will open another PR with the changes if this approach is fine.

Reply:
@siddhsuresh Ya that's an option. Wondering why esbuild-register wouldn't work with compiling js files though.